classes/author.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require_relative 'item'
+
+# Author class
+class Author
+  attr_accessor :name, :last_name
+  attr_reader :items
+
+  def initialize(first_name, last_name)
+    @name = first_name
+    @last_name = last_name
+    @items = []
+  end
```

Comment:
Please consider adding the `id` variable in class Author as indicated in the UML diagram [click here for UML >>](https://github.com/microverseinc/curriculum-ruby/raw/main/group-capstone/images/catalog_of_my_things.png)